test/keepAliveTimeout.js

```diff
@@ -1,24 +1,22 @@
 const assert = require("assert");
```

Comment:
Why is the file a part of this PR?

Reply:
Looks like a `git mv` on an incorrect branch while plowing through a bunch updates. This can be restored.